base/exports.jl

```diff
@@ -179,18 +182,26 @@ export
     *,
     +,
     -,
+    −,
```

Comment:
I think we already handle U+2212 with our Unicode normalization?   See #40948

Reply:
Yeah, it is true, I read that some months ago in a release note!